BraintreeCore/src/main/java/com/braintreepayments/api/core/ApiClient.kt

```diff
@@ -14,17 +13,15 @@ class ApiClient(private val braintreeClient: BraintreeClient) {
     fun tokenizeGraphQL(tokenizePayload: JSONObject, callback: TokenizeCallback) =
         braintreeClient.run {
             sendAnalyticsEvent("card.graphql.tokenization.started")
-            sendGraphQLPOST(tokenizePayload, object : HttpResponseCallback {
-                override fun onResult(responseBody: String?, httpError: Exception?) {
-                    parseResponseToJSON(responseBody)?.let { json ->
-                        sendAnalyticsEvent("card.graphql.tokenization.success")
-                        callback.onResult(json, null)
-                    } ?: httpError?.let { error ->
-                        sendAnalyticsEvent("card.graphql.tokenization.failure")
-                        callback.onResult(null, error)
-                    }
+            sendGraphQLPOST(tokenizePayload) { responseBody, httpError ->
+                parseResponseToJSON(responseBody)?.let { json ->
+                    sendAnalyticsEvent("card.graphql.tokenization.success")
```

Comment:
Are these also leftover events from v4?

Reply:
🤔 I'm not sure if the v5 core events were ever updated. I'm only seeing a latency event defined in the `CoreAnalytics` class:

```kotlin
internal object CoreAnalytics {
    const val apiRequestLatency = "core:api-request-latency"
}
```

Reply:
Ok I think in an effort to reduce analytics noise we should go ahead and remove any events not defined in our shared iOS/Android events list (which should all existing in the Analytics enums now)

Reply:
Sounds good. I'll remove these events in this PR and follow up with an audit of all events sent.

Reply:
Updated: 0d389595336d6656ab98c0148f80cab36db5307b